### ISSUE TITLE
Deprecate events api

### ISSUE THREAD

**Issue by dmichel1** [spotify/ffwd#130]
Currently the event support of ffwd is not used internally within Spotify and there are no plans to. Does it make sense to continue supporting it before we cut a major release of ffwd?

**Comment by hexedpackets** [comment]
Its also experimental/unfinished on the Heroic side of things.

**Comment by hexedpackets** [comment]
Since we've decided to remove events from Heroic (https://github.com/spotify/heroic/issues/479) we should also remove them from ffwd.

**Comment by hexedpackets** [comment]
To fully remove this we'll have to also remove the riemann input, since that exclusively outputs events. 

@dmichel1 do you think we can do that now or do we need a slower deprecation? I don't think internally we're using that input anywhere. So maybe after #120 is done we can remove it.

**Comment by dmichel1** [comment]
@hexedpackets are we 100%  on removing event support as a concept from ffwd? this decision can be made independently from heroic, as I don't think heroic is a good fit for this type of data. 

I could see a future where we want to have a more robust pipeline of events that go to elasticsearch or a similar database. Whether or not we want to funnel that through ffwd is the bigger question.

**Comment by hexedpackets** [comment]
Events have never been fully supported - kafka and riemann were the only outputs that allowed them, everything else just drops events. I also think events are different enough from metrics that it doesn't make sense to funnel them through the same pipeline. Events are closer to logs with a need for stronger delivery guarantees. Guaranteed delivery isn't a goal of ffwd, and many of the interesting things we can do around metrics wouldn't apply to events or would have to be implemented very differently.

tl;dr: yes let's remove it

**Comment by dmichel1** [comment]
@hexedpackets sgtm - lets remove it!
